components/eam/src/chemistry/pp_linoz_mam4_resus_mom_soag/mo_setrxt.F90

```diff
@@ -9,32 +9,27 @@ module mo_setrxt
 
       contains
 
-      subroutine setrxt( rate, temp, m, ncol )
+      subroutine setrxt( rate, temp, ncol )
 
       use ppgrid,       only : pver, pcols
       use shr_kind_mod, only : r8 => shr_kind_r8
       use chem_mods, only : rxntot
-      use mo_jpl,    only : jpl
 
       implicit none
 
 !-------------------------------------------------------
 !       ... dummy arguments
 !-------------------------------------------------------
-      integer, intent(in) :: ncol
-      real(r8), intent(in)    :: temp(pcols,pver)
-      real(r8), intent(in)    :: m(ncol,pver)
-      real(r8), intent(inout) :: rate(ncol,pver,rxntot)
+      integer, intent(in) :: ncol    ! number of columns
+      real(r8), intent(in)    :: temp(pcols,pver)  ! midpoint temperature [K]
+      real(r8), intent(inout) :: rate(ncol,pver,rxntot) ! reaction rate 
 
 !-------------------------------------------------------
 !       ... local variables
 !-------------------------------------------------------
-      integer  ::  n
-      real(r8)  ::  itemp(ncol,pver)
-      real(r8)  ::  exp_fac(ncol,pver)
+      real(r8)  ::  itemp(ncol,pver)  ! inverse midpoint temperature [K^-1]
 
       itemp(:ncol,:) = 1._r8 / temp(:ncol,:)
-      n = ncol*pver
       rate(:,:,3) = 2.9e-12_r8 * exp( -160._r8 * itemp(:,:) )
       rate(:,:,5) = 9.6e-12_r8 * exp( -234._r8 * itemp(:,:) )
       rate(:,:,7) = 1.9e-13_r8 * exp( 520._r8 * itemp(:,:) )
```

Comment:
`setrxt_hrates` below seems empty. if it is, please remove it.

Reply:
Done.